lib/config-generator.js

```diff
@@ -166,22 +167,22 @@ const compat = new FlatCompat({baseDirectory: __dirname, recommendedConfig: plug
         }
 
         if (this.answers.language === "typescript") {
+            extensions.push("ts");
             this.result.devDependencies.push("typescript-eslint");
             this.result.installFlags.push("--force"); // tseslint does not support eslint 9.x yet
             importContent += "import tseslint from \"typescript-eslint\";\n";
             exportContent += "  ...tseslint.configs.recommended,\n";
         }
 
         if (this.answers.framework === "vue") {
-
+            extensions.push("vue");
             this.result.devDependencies.push("eslint-plugin-vue");
-
             importContent += "import pluginVue from \"eslint-plugin-vue\";\n";
             exportContent += "  ...pluginVue.configs[\"flat/essential\"],\n";
         }
 
         if (this.answers.framework === "react") {
-            exportContent += "  { files: [\"**/*.jsx\"], languageOptions: { parserOptions: { ecmaFeatures: { jsx: true } } } },\n";
+            extensions.push("jsx");
```

Comment:
its would be better to move the `react + ts` condition inside here?

Reply:
i like it 👍. updated.